component-playground/config.js

```diff
@@ -24,10 +26,15 @@ try {
   }
 }
 
-var webpackConfig = {
+config.server = _.extend({
+  port: 8989,
+  hostname: 'localhost'
+}, userConfig.server);
+
+config.webpack = {
   context: playgroundPath,
   entry: [
-    'webpack-dev-server/client?http://localhost:8989',
+    'webpack-dev-server/client?http://localhost:' + config.server.port,
```

Comment:
Sorry, I just don't have the whole thing at home, will try to clone your cosmos-example project and test with it